use-timescale/hypertables/create.md

```diff
@@ -9,8 +9,8 @@ keywords: [hypertables, create]
 
 <Highlight type="note">
 This code uses the new generalized hypertable API introduced in
-TimescaleDB 2.13. The [old interface for `create_hypertable is also
-available](create_hypertable_old.md).
+TimescaleDB 2.13. The [old interface for `create_hypertable` is also
+available][create-hypertable-old].
```

Comment:
This link is still broken. I think these types of links don't work in Highlight blocks.
```suggestion
available](/api/:currentVersion:/hypertable/create_hypertable_old/).
```